docs/index.md

```diff
@@ -31,7 +31,7 @@ Before using Terraform for creating resources in LogDNA, review the following no
 - Verify Terraform is [installed](https://learn.hashicorp.com/tutorials/terraform/install-cli). The minimum supported version is 0.12.0 and can be checked by running `terraform version`.
 - Have the service key for your Organization available. To obtain the service key for your LogDNA Organization, go to the LogDNA dashboard and navigate to **Settings > Organization > API Keys** or follow this link [here](https://app.logdna.com/manage/api-keys).
 - Authentication is handled via the `servicekey` parameter and can be set in the provider configuration.
```

Comment:
@tanberry this looks good!! would add "This PR is for the edits to the index.md file, per feedback from Support team." to the commit message as well along with the Ref and the Semver.

Additionally it looks like we forgot to add jsonencode to the bodytemplate in the README and the view.md as we did in the examples. Could we incorporate that change to the below two files:

https://github.com/logdna/terraform-provider-logdna/blob/main/README.md
https://github.com/logdna/terraform-provider-logdna/blob/main/docs/resources/logdna_view.md

Jsonencode in the examples:
https://github.com/logdna/terraform-provider-logdna/blob/main/examples/view_with_email_pagerduty_and_webhook_alerts.tf

Reply:
The other thing is we need to rebase with main- with `git rebase main`